0006-cmx2cdp-protocol/README.md

```diff
@@ -1,84 +1,102 @@
 ---
 name: CMX to CDP protocol
 shortname: 6/CMX2CDP
-status: raw
-editor: Andrew Pasika <apasika@cybervisiontech.com>
+status: draft
+editor: Andrew Pasika<apasika@cybervisiontech.com>
```

Comment:
With spaces, it results in redundant space before comma: "Alexey Gamov , Andrew Kokhanovskyi". But @rasendubi says I should bring back the space to avoid potential rendering issues, so I'll bring back the spaces.